doctr/models/kie_predictor/pytorch.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (C) 2021-2022, Mindee.
```

Comment:
I agree with you that a lot of things are duplicated for now. But as there is no breaking change, it's something that we can work on after this is released and make it better. ( remove the redundancies, maybe have a abstract predictor more general and inherit from it, ...) 

For the fact that the PR is too big and opening dicsussion/issue about some part, unfortunately we don't have time to do that. We want the multiclass feature to be available as soon as possible, even if it's not perfect and improve it after but at least have it running.

Reply:
Mh .. i agree if you want to use it "now" but sometimes it is better to take the time instead of brute forcing things :sweat_smile: the refactoring will take some more time as doing it now in an better way and i am sure it will break again things which are done in your PR. :thinking: 

Unfortunately, I don't see the PR in good condition right now, which is why I wouldn't / can't approve it, but in the end this is not my decision, if you have discussed this internally (@odulcy-mindee @charlesmindee).